core/src/main/scala/kafka/coordinator/group/CoordinatorLoaderImpl.scala

```diff
@@ -143,10 +150,12 @@ class CoordinatorLoaderImpl[T](
 
               currentOffset = batch.nextOffset
             }
+            numBytes = numBytes + memoryRecords.sizeInBytes()
           }
+          val endTimeMs = time.milliseconds()
 
           if (isRunning.get) {
-            future.complete(null)
+            future.complete(new LoadSummary(startTimeMs, endTimeMs, numRecords, numBytes))
```

Comment:
should we include information about the LoadSummary in the PR description as well?